qt/applications/workbench/workbench/plugins/memorywidget/memoryinfo.py

```diff
@@ -0,0 +1,20 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+#  This file is part of the mantidqt package
+#
+#
+from psutil import virtual_memory
+conversion_factor_to_GB = 1.0 / (1024 * 1024 * 1024)
```

Comment:
Good to see a named constant. Can we plase have all capitals for module constants and a clear blank line to separate the imports from the constants?